Showcase-iOS/About/View/AboutViewController.swift

```diff
@@ -32,44 +24,49 @@ class AboutViewController: UIViewController {
         aboutPresenter?.retrieveSocialMediaLinks()
     }
 
-    // MARK: Opertaions
+    // MARK: Opertaion(s)
 
     func setupInjectables() {
         let aboutPresenter = dependencyContainer.resolve(AboutPresentable.self) as? AboutPresenter
         let firebaseStorage = dependencyContainer.resolve(FIRStoring.self)
         self.aboutPresenter = aboutPresenter
         aboutPresenter?.aboutView = self
+        aboutPresenter?.analyticManager = dependencyContainer.resolve(AnalyticsManager.self) as? AnalyticManagerImplementation
         self.firebaseStorage = firebaseStorage
     }
 
     // MARK: @IBActions
 
     @IBAction func websiteTapped(_ sender: Any) {
+        aboutPresenter?.trackSocialMediaButtonTap(with: "website")
         if let websiteUrl = aboutViewModel?.website {
             UIApplication.shared.open(websiteUrl, options: [:], completionHandler: nil)
         }
     }
 
     @IBAction func twitterTapped(_ sender: Any) {
+        aboutPresenter?.trackSocialMediaButtonTap(with: "twitter")
```

Comment:
Can we not make this an String enum and use it through there as this is very hardcoded. We can put all the analytics strings in those enums and then they can be referenced from one place.

Reply:
Will do so now :)